tools/benchmark-cli/src/main/java/org/logstash/benchmark/cli/LogstashInstallation.java

```diff
@@ -46,12 +47,17 @@
 
         private final LogstashInstallation base;
 
-        public FromRelease(final File pwd, final String version) {
+        public FromRelease(final File pwd, final String version, final UserOutput output) {
             try {
-                LogstashInstallation.FromRelease.download(pwd, version);
-                this.base = LogstashInstallation.FromRelease.setup(
-                    pwd.toPath().resolve(String.format("logstash-%s", version))
-                );
+                final Path basedir = pwd.toPath().resolve(String.format("logstash-%s", version));
+                if (basedir.toFile().exists()) {
+                    output.blue(String.format("Using Logstash %s from cache.", version));
+                } else {
+                    output.blue(String.format("Downloading Logstash %s.", version));
+                    LogstashInstallation.FromRelease.download(pwd, version);
+                    output.blue("Finished downloading Logstash.");
```

Comment:
@jakelandis seems I didn't do such a good job here quality wise :(, doing that is a lot (relative to the benefit) of additional code it seems.
Let's improve the UI here in the next iteration?